src/main/java/com/amplitude/Amplitude.java

```diff
@@ -59,6 +61,13 @@ public void init(String key) {
     apiKey = key;
   }
 
+  public void isBatchMode(Boolean isBatchMode) {
```

Comment:
the method name is confusing though, by `isBatchMode` it sounds to be a method checking if it is batch mode, other than trying to set to use batch.